llvm/include/llvm/BinaryFormat/Dwarf.def

```diff
@@ -565,6 +567,13 @@ HANDLE_DW_AT(0x2303, GNU_numerator, 0, GNU)
 HANDLE_DW_AT(0x2304, GNU_denominator, 0, GNU)
 HANDLE_DW_AT(0x2305, GNU_bias, 0, GNU)
 
+// AIE
+HANDLE_DW_AT(0x2501, AIE_frame_size, 0, AMD)
+HANDLE_DW_AT(0x2502, AIE_frame_base, 0, AMD)
+HANDLE_DW_AT(0x2503, AIE_addr, 0, AMD)
+HANDLE_DW_AT(0x2504, AIE_frame_base2, 0, AMD)
```

Comment:
UPSTREAM WITH YOUR BACKEND: It's been a while since I looked at DWARF but the one comment I have is I would remember to use existing attributes in possible to make your life easier with your debugger and binary tools in general. That may not be possible though.